core/trino-main/src/test/java/io/trino/operator/output/TestPagePartitioner.java

```diff
@@ -478,22 +483,48 @@ private PagePartitionerBuilder pagePartitioner()
 
     private enum PartitioningMode
     {
-        ROW_WISE {
+        ROW_WISE(true) {
             @Override
             public void partitionPage(PagePartitioner pagePartitioner, Page page)
             {
                 pagePartitioner.partitionPageByRow(page);
             }
         },
-        COLUMNAR {
+        COLUMNAR(true) {
             @Override
             public void partitionPage(PagePartitioner pagePartitioner, Page page)
             {
                 pagePartitioner.partitionPageByColumn(page);
             }
+        },
+        DEFAULT_WITH_COLUMNAR_ON(true) {
```

Comment:
I think we can simplify this test class and just create `PagePartitioner` with `acceleratedRepartitioningEnabled` enabled/disabled.

Are `DEFAULT_WITH_COLUMNAR_ON|OFF` testing some other scenario?